multiuser/permission/che-multiuser-permission-workspace/pom.xml

```diff
@@ -25,6 +25,10 @@
         <findbugs.failonerror>false</findbugs.failonerror>
     </properties>
     <dependencies>
+        <dependency>
+            <groupId>aopalliance</groupId>
+            <artifactId>aopalliance</artifactId>
```

Comment:
I think test scope must be added, otherwise it won't be possible to compile this module without tests compiling.